chain/chain/src/chain.rs

```diff
@@ -366,11 +366,11 @@ impl Chain {
             pending_state_patch: Default::default(),
             requested_state_parts: StateRequestTracker::new(),
             snapshot_callbacks: None,
-            state_split_config: MutableConfigValue::new(
-                StateSplitConfig::default(),
-                "state_split_config",
```

Comment:
Same here, confirming if we can safely change "state_split_config" to "resharding_config"

Reply:
same as above, resolving